Update of 302 and 305 NNCF notebooks. Logging and metrics.

negvet opened the pull request:
- Remove logging of INFO and WARNING levels (warning may confuse users, basically all actions are explicitly commented in the appropriate cells). 
- Update output metrics in TF to be aligned.

helena-intel commented:
Thanks @negvet it is very welcome to have fewer warning messages about things that are not actually a concern. I prefer not to blanket hide all warnings though - presumably these warnings are there for a reason? I'm also not against hiding all NNCF warnings in the notebook if the alternatives are uglier, but then we should have a comment above that that explains that NNCF displays some warnings that can be ignored for the purposes of the tutorial - that hopefully helps with people who blindly copy/paste all the imports.

It also seems not ideal to hide all warnings - maybe other libraries do show warnings that we want to see. But I still see warnings with this `warnings.disable` call. Is that expected?
![image](https://user-images.githubusercontent.com/77325899/144615459-361bb5f4-3945-416e-87c7-d404aa623814.png)

negvet commented:
@helena-intel thanks for the feedback.
I updated the notebooks so that only NNCF-related logging is disabled. Note is also added.
Please check it out.

helena-intel commented on notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb:
````diff
@@ -35,7 +35,9 @@
-    "Set a name for the model, and the image width and height that will be used for the network. Also define paths where PyTorch, ONNX and OpenVINO IR versions of the models will be stored. "
+    "Set a name for the model, and the image width and height that will be used for the network. Also define paths where PyTorch, ONNX and OpenVINO IR versions of the models will be stored. \n",
+    "\n",
+    "> NOTE: All NNCF logging messages below ERROR level (INFO and WARNING) are disbaled to simplify the tutorial. For real usage scenario it is recommended to enable logging, by removing ```set_log_level(logging.ERROR)```."
````
```suggestion
    "> NOTE: All NNCF logging messages below ERROR level (INFO and WARNING) are disabled to simplify the tutorial. For production use, it is recommended to enable logging, by removing ```set_log_level(logging.ERROR)```."
```

helena-intel commented on notebooks/305-tensorflow-quantization-aware-training/305-tensorflow-quantization-aware-training.ipynb:
````diff
@@ -24,9 +24,10 @@
-    "Set a name for the model, input image size, used batch size, and the learning rate. Also define paths where Frozen Graph and OpenVINO IR versions of the models will be stored."
+    "Set a name for the model, input image size, used batch size, and the learning rate. Also define paths where Frozen Graph and OpenVINO IR versions of the models will be stored.\n",
+    "\n",
+    "> NOTE: All NNCF logging messages below ERROR level (INFO and WARNING) are disbaled to simplify the tutorial. For real usage scenario it is recommended to enable logging, by removing ```set_log_level(logging.ERROR)```."
````
```suggestion
    "> NOTE: All NNCF logging messages below ERROR level (INFO and WARNING) are disabled to simplify the tutorial. For production use, it is recommended to enable logging, by removing ```set_log_level(logging.ERROR)```."
```

helena-intel reviewed (changes requested):
Thanks! I have a minor language suggestion, otherwise it looks good. I still see this warning: 
![image](https://user-images.githubusercontent.com/77325899/145103481-9e2f68b5-adcb-482f-b133-cad0384bac20.png)
I suppose this is because of warnings instead of logging, and the note is clear about that. Are we missing important NNCF functionality by using PyTorch 1.7? I would like to update to a higher version of PyTorch, but the 102 notebook currently breaks with a higher version.

negvet commented:
> Are we missing important NNCF functionality by using PyTorch 1.7?

@ljaljushkin might you please advise here?

ljaljushkin commented:
> > Are we missing important NNCF functionality by using PyTorch 1.7?
> 
> @ljaljushkin might you please advise here?

AFAIK, we don't test nncf with all previous versions of pytorch, that's why we don't guarantee best results with them.
nncf is validated with torch1.9.1.

negvet commented:
Jenkins please retry a build

helena-intel commented:
Thanks @negvet !
